app/services/find_target_user.py

```diff
@@ -15,15 +16,15 @@
 logger = Logger(__name__)
 
 
-def get_target_user(message: types.Message, can_be_same=False, can_be_bot=False) -> typing.Optional[types.user.User]:
+def get_target_user(message: types.Message, can_be_same=False, can_be_bot=False) -> typing.Optional[dto.TargetUser]:
```

Comment:
We need to choose. Optional as `x | None` or `Optional[x]`. Currently it seems inconsistent

Reply:
I was fix this file, but all codebase is too large. I think this changes too much increase git diff for this PR